static/js/gtm.js

```diff
@@ -0,0 +1,47 @@
+window.dataLayer = window.dataLayer || [];
```

Comment:
We should probably put a license header here (+ on gtag.js)

Reply:
Good idea :D